sql/moz-fx-data-shared-prod/telemetry/nondesktop_clients_last_seen_v1/view.sql

```diff
@@ -34,7 +34,7 @@ WITH glean_final AS (
     country,
     locale,
     app_display_version,
-    'Lockwise Baseline' AS app_name,
+    'Lockwise Android' AS app_name,
```

Comment:
These Baseline names were introduced for apps that are also present in `core` pings. We may introduce double-counting of clients from Glean and legacy telemetry by making these name changes. Focus is one of those apps.

Reply:
I see. Would it make sense to ignore Focus/Klar clients from `core` pings by filtering them out here: https://github.com/mozilla/bigquery-etl/blob/98549e3cb8b4f81d03a08a33aedaf6687ea5f769/sql/moz-fx-data-shared-prod/telemetry/nondesktop_clients_last_seen_v1/view.sql#L223 and instead use their Glean data?